script.js

```diff
@@ -358,12 +358,22 @@ closePopup();
 // form validation
 
 const form = document.getElementById('contact');
+const username = document.getElementById('name');
 const email = document.getElementById('email');
+const message = document.getElementById('textBody');
 const errormsg = document.getElementById('error');
 
 form.addEventListener('submit', (e) => {
   if (email.value.toLowerCase() !== email.value) {
     errormsg.style.display = 'block';
     e.preventDefault();
   }
-});
\ No newline at end of file
+
+  localStorage.setItem('name-input', username.value);
+  localStorage.setItem('email-input', email.value);
+  localStorage.setItem('msg-input', message.value);
+});
+
+username.value = localStorage.getItem('name-input');
+email.value = localStorage.getItem('email-input');
+message.value = localStorage.getItem('msg-input');
```

Comment:
Great job storing the values after submitting the form! Although, as requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/202735197-59bfa015-d912-45ff-b6bb-2c9234d9d97a.png)

Kindly notice that the user data should be stored after any changes to the form, not after submitting it 😉.